src/v2/Components/NavBar/Menus/MoreNavMenu.tsx

```diff
@@ -20,10 +20,8 @@ export const MoreNavMenu: React.FC<{ width?: number }> = ({ width = 160 }) => {
   }
 
   return (
-    <Menu onClick={trackClick} width={width}>
-      {/*
-        Hide nav items at md / lg as they appear in the top nav
-      */}
+    <Menu onClick={trackClick} width={width} m={0}>
```

Comment:
Is there a default inner `m` on Menu? If not, can we remove `m={0}`?

Reply:
Yeah there is. Which honestly is weird/uncool: https://github.com/artsy/force/blob/e941daca66eb7470d0b6858842525c384b49b4eb/src/v2/Components/Menu.tsx#L31